content/applications/inventory_and_mrp/quality/quality_reporting.rst

```diff
@@ -0,0 +1,10 @@
+:nosearch:
+
+=================
+Quality reporting
```

Comment:
I totally get that the portion of this app is known as 'Reporting' -- so it probably doesn't come into play here -- but, in the past, I've been instructed to avoid using -ing words as section/doc headers. So, perhaps, it might be best to consider rewording this title to 'Quality reports'...? Not a requirement, just a suggestion for something to consider during revisions.